src/store/channels-list/saga.ts

```diff
@@ -63,19 +63,26 @@ export function* mapToZeroUsers(channels: any[]) {
   return;
 }
 
-export function* updateOtherMembersLastSeenAt(conversations) {
+export function* updateUserPresence(conversations) {
   if (!featureFlags.enableMatrix) {
     return;
   }
 
   const chatClient = yield call(chat.get);
   for (let conversation of conversations) {
-    const matrixId = conversation?.otherMembers?.[0]?.matrixId;
+    const { isOneOnOne, otherMembers } = conversation;
+    const matrixId = otherMembers?.[0]?.matrixId;
 
-    if (conversation.isOneOnOne && matrixId) {
+    if (isOneOnOne && matrixId) {
```

Comment:
Don't we still show the green dot on group conversations?

Reply:
Yeah actually I don't think this will won't work for group conversations. I'll need to take another look.

Reply:
Nice, Updated this!

<img width="720" alt="Screenshot 2023-10-17 at 13 12 44" src="https://github.com/zer0-os/zOS/assets/39112648/beca7a83-1822-433f-b03f-1e9feb99f306">